src/components/v2v3/V2V3Project/ProjectSafeDashboard/ProjectSafeDashboard.tsx

```diff
@@ -81,28 +82,41 @@ export function ProjectSafeDashboard() {
         </ExternalLink>
       ) : null}
 
-      {isLoading && <div style={{ marginTop: 20 }}>Loading...</div>}
+      {isQueuedLoading && <div style={{ marginTop: 20 }}>Loading...</div>}
 
-      {!isLoading && (
+      {!isQueuedLoading && (
         <div style={{ marginTop: '1.5rem' }}>
-          <Tab
-            name={TAB_NAMES.queued}
-            isSelected={selectedTab === SAFE_TX_QUEUED_KEY}
-            onClick={() => setSelectedTab(SAFE_TX_QUEUED_KEY)}
-          />
+          <Space size="large">
+            <Tab
```

Comment:
**suggestion (non-blocking)**: we should update this and the Projects page to use antd tabs instead probably (and restyle them)